src/toolkitAPI.c

```diff
@@ -1481,3 +1474,41 @@ int DLLEXPORT swmm_setOutfallStage(int index, double stage)
     }
     return(errcode);
 }
+
+int DLLEXPORT swmm_setGagePrecip(int index, double value)
```

Comment:
The getter returns rain, snow, and total while the setter only set rain. This seems odd?

Reply:
@michaeltryby the type of precipitation is a function of temperature.  If it is below the snow point the internal mechanics do the allocation.  The counterpart value for this function in the get_precip is "total precip"

Reply:
Then wouldn't it be more clear to name the variable returned "total precip" instead of value?